src/test/java/teammates/test/cases/browsertests/FeedbackRubricQuestionUiTest.java

```diff
@@ -68,38 +68,8 @@ private void testInstructorResultsPage() throws Exception {
         InstructorFeedbackResultsPage instructorResultsPage =
                 loginToInstructorFeedbackResultsPageWithViewType("teammates.test.instructor", "openSession2",
                                                                  false, "question");
-        instructorResultsPage.waitForPanelsToExpand();
-
+        clickAjaxLoadedPanelAndWaitForExpansion(instructorResultsPage, "panelHeading-1", "ajax_auto");
         instructorResultsPage.verifyHtmlMainContent("/instructorFeedbackResultsPageRubricQuestionView.html");
-
-        // Giver Recipient Question View
-        instructorResultsPage =
```

Comment:
Can we address this? I understand that RGQ, GRQ etc are tested on ```FeedbackRankQuestionUiTest.java``` and I want to make sure that they are meant for testing the same thing and that's why we are removing them..

1. If they are testing the same thing, we can remove them entirely, the name does indicate some difference so we should make sure, one is feedback ui, the other is rubric so we need to investigate if they are the same thing

2. If they are not testing the same thing, the solution here should be to still continue testing all these and remove the AJAX portion, i.e. test the HTML but no need to wait for panels to expand